### PR TITLE
Update MapboxCoreSearch v2.5.1, Release v2.5.1

### PR DESCRIPTION
**Pull request by aokj4ck**
### Description

- Update MapboxCoreSearch v2.5.1
- Prepare release for v2.5.1

### Checklist
- [x] Update `CHANGELOG`

### CODE REVIEW

File: MapboxSearch.podspec [hunk 1]
```diff
@@ -24,6 +24,6 @@ Some iOS platform specifics applies.
 
   s.vendored_frameworks = "**/#{s.name}.xcframework"
 
-  s.dependency "MapboxCoreSearch", '2.5.0'
+  s.dependency "MapboxCoreSearch", '2.5.1'
```
Comment (kried) [inline MapboxSearch.podspec:27]: Was this pull created in advanced? [I don't see MapboxCoreSearch v2.5.1 release in CocoaPods](https://github.com/CocoaPods/Specs/tree/master/Specs/6/8/5/MapboxCoreSearch).

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,9 +8,14 @@ Guide: https://keepachangelog.com/en/1.0.0/
 
 <!-- Add changes for active work here -->
 
+## 2.5.1
```
Comment (kried) [inline CHANGELOG.md:11]: I was expecting the 2.6.0-beta.1 release with Common 24.8.0-beta.1. Do you plan to release both, v2.5.1 and v2.6.0-beta.1?

Comment (aokj4ck) [reply CHANGELOG.md:11]: Yes

File: MapboxSearch.podspec (hunk 1 above)
Comment (aokj4ck) [reply MapboxSearch.podspec:27]: Yes

Comment (aokj4ck) [reply MapboxSearch.podspec:27]: Yes pod has been published

**Review by kried — APPROVED** [review]

**Pull request merged**